### ISSUE TITLE
 [BUG/TYPO] Accessibility issues for non-visual access

### ISSUE THREAD

**Issue by funkonaut** [Robbbb/VectorRuler#35]
**Describe the bug**
The site has issues that hinder non-visual navigation with a screen reader.

**To Reproduce**
1. Open screen reader of choice (ie [NVDA](https://www.nvaccess.org/download/), Windows Narrator (control + Windows key + enter), VoiceOver (command + f5 or power button 3 times fast)), ChromeVox (hold CVOX key (typically alt or control) A A in quick succession)
2. Focus the web application.
3. Tab between elements notice how some elements are read aloud but some are not. 

**Expected behaviour**
All input elements should be appropriately labeled to be read aloud by screen readers.

**Screenshots**
n/a

**Desktop (please complete the following information if applicable):**
 - OS: all
 - Browser: all
 - Version: all

**Smartphone (please complete the following information if applicable):**
 - Device: iOS, android
 - OS: all
 - Browser: all
 - Version: all

**Additional context**
My pull request attempts to resolve the following which should enable non-visual users to access the website:
1. Update README to provide more descriptive alt text for link to web app
2. Label sub-unit radio buttons with a visual label
4. Add a non-visual label (aria-label) to redundant lines checkbox 
5. Fix the Label for PPI so it is associated with the corresponding radio button input field
6. Add line break to left justify PPI radio buttons

**Comment by darigovresearch** [comment]
Should be resolved from merged pull request #36 

@funkonaut thanks for raising this & making the fix!
